base/db/DESCRIPTION

```diff
@@ -25,7 +25,12 @@ Imports:
     udunits2
 Suggests:
     RPostgreSQL,
-    testthat (>= 1.0.2)
+    RSQLite,
+    testthat (>= 1.0.2),
+    tidyverse,
```

Comment:
Point taken, but it is only a `Suggests` rather than a `Imports/Depends`, and this way, it matches with what I'm doing in the scripts. Though, grudgingly, I'll admit it is good coding practice to be as explicit as possible about where things are coming from.

In general, my approach is to stick packages required for scripts but not internal functions in `Suggests`, since I think those aren't installed unless you explicitly ask for them in `devtools::install` (right?).

Also, as an aside, get a load of the [`import` package](https://cran.r-project.org/web/packages/import/vignettes/import.html), which brings pythonic `from Z import X as Y` syntax to R. I might switch to using this in a bunch of my interactive scripts.

Reply:
Fair point on suggests vs imports, but I see calls to all three of tibble, purrr, tidyr in `R/`, so those should probably be in imports anyway 😉